### DISCUSSION TITLE
Stop Motion OBJ showcase

### DISCUSSION THREAD

**Discussion by neverhood311** [neverhood311/Stop-motion-OBJ#118] [Show and tell]
I want a place for people to show off what they've made with Stop Motion OBJ. Maybe we can inspire one another to create beautiful things. It also challenges me to constantly improve the features, stability, and user experience.

Reply to this thread with your work. Please include links and thumbnails, if possible. Also feel free to post other people's public work (please don't post private/unlisted work from other people unless you've received permission from the artist to do so).

**Comment by neverhood311** [comment]
[![thing](https://img.youtube.com/vi/ZuzjzkTnv9s/0.jpg)](https://www.youtube.com/watch?t=141&v=ZuzjzkTnv9s)
[Le Salon des Nobles de la Reine](https://www.youtube.com/watch?t=141&v=ZuzjzkTnv9s) by SoftYoda. The violin and cello players are Kinect point clouds, imported as a mesh sequence.

**Reply by softyoda** [reply]
Thanks for the showcase ! 
This add-on helped me a lot to get this sequence from kinect into blender !

**Comment by ThomasSchumacher77** [comment]
Velocity contours around vehicles driving in a convoy. Detached Eddy Simulation with ELEMENTS-CFD.
Horizontal planes exported from Paraview as ply files.
[![ALT Text](https://i1.wp.com/blog.engys.com/wp-content/uploads/2021/02/blogThumbnail.png?resize=820%2C510&ssl=1)](https://www.youtube.com/watch?v=FcRJDfnbAMQ)
Learn more about the case [in my Blog post](https://blog.engys.com/power-in-numbers-assessing-vehicle-convoy-aerodynamics-with-cfd-simulation/)
